src/modules/user/UserPage.tsx

```diff
@@ -86,15 +106,45 @@ export class UserPage extends React.Component<Props, State> {
                       </a>
                     </span>
                     {data && data.viewer.slackId ? (
-                      <Header data-testid="slack-connected">Your account is connected to slack!</Header>
+                      <Segment data-testid="slack-connected">
```

Comment:
Maybe you could extract this whole Segment into a separate component or private function for readability? The render function is pretty big now.

Reply:
Will do!